WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/HistoryItemViewModelBase.cs

```diff
@@ -52,6 +54,8 @@ public virtual void Update(HistoryItemViewModelBase item)
 		Date = item.Date;
 		DateString = item.DateString;
 		IsConfirmed = item.IsConfirmed;
+		Label = item.Label;
+		FilteredLabel = item.FilteredLabel;
```

Comment:
Even if this does not cause any issue, it is not correct here. Because it suggests that the labels are updated in any case, but if it is a CoinjoinGroup, the labels don't need to be updated.

Can you move this to the `TransactionHistoryItemViewModel.cs`?